### PR TITLE
fresh commit for ndpi project

### PR DESCRIPTION
**Pull request by headshog**

### CODE REVIEW

**Comment by koltiradw** [comment]
Hi! U can also add [fuzz_tls_certificate](https://github.com/ntop/nDPI/blob/dev/fuzz/fuzz_tls_certificate.c) target.

**Review by SweetVishnya — CHANGES_REQUESTED** [review]
@headshog, can you add new targets as @koltiradw suggested?

**Comment by headshog** [comment]
> @headshog, can you add new targets as @koltiradw suggested?

Yes, okay

**Review by anfedotoff — APPROVED** [review]

**Review by kobrineli — CHANGES_REQUESTED** [review]

File: projects/ndpi/load_sydr_tls_certificate.c [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+        memset(ndpi_flow, 0, sizeof(struct ndpi_flow_struct));
+        strcpy(ndpi_flow->host_server_name, "doh.opendns.com");
+        ndpi_flow->detected_protocol_stack[0] = NDPI_PROTOCOL_TLS;
+
+        processCertificateElements(ndpi_struct, ndpi_flow, 0, size);
+        ndpi_free_flow_data(ndpi_flow);
+
+        return 0;
+}
```
[the remark this answers is not in the document]
Comment (headshog) [reply projects/ndpi/load_sydr_tls_certificate.c]: done

**Pull request merged**